.env.test

```
@@ -17,6 +17,10 @@ GITHUB_CLIENT_ID=123;
 GITHUB_CLIENT_SECRET=123;
 GITHUB_APP_NAME=outline-test;
 
+GITLAB_URL=http://localhost
```

```suggestion
GITLAB_URL=
```

Better to leave empty in test I would think